README.md

```diff
@@ -4,59 +4,84 @@
 
 ![Screenshot](.github/screenshot.png?raw=true)
 
-
 # Usage
 
-## Dependencies
+## With Docker Compose
+
+The easiest way to run Gitbase Playground and its dependencies is using [Docker Compose](https://docs.docker.com/compose/install/).
 
-The playground will run the queries against a [gitbase](https://github.com/src-d/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server; both should be accessible for the playground; you can check its default [configuration values](docs/CONTRIBUTING.md#configuration).
+The first step is to populate a directory with some Git repositories to be served by gitbase before running it. For example:
 
+```bash
+$ mkdir $HOME/repos
+$ cd $HOME/repos
+$ git clone git@github.com:src-d/gitbase.git
+$ git clone git@github.com:bblfsh/bblfshd.git
+$ git clone git@github.com:src-d/gitbase-playground.git
+```
 
-## Run the Playground
+Next you will need to download the `docker-compose.yml` file from this repository and run `docker-compose`. This tool will run three different containers: the playground frontend itself, gitbase, and bblfshd. To kill the running containers use `Ctrl+C`.
+
+```bash
+$ wget https://raw.githubusercontent.com/src-d/gitbase-playground/master/docker-compose.yml
+$ docker-compose pull
+$ GITBASEPG_REPOS_FOLDER=$HOME/repos docker-compose up --force-recreate
+```
 
-You can run the app from a docker image, a released binary or installing and building the project.
+The server should be now available at http://localhost:8080.
 
-Once the server is running &ndash;with its default values&ndash;, it will be accessible through: http://localhost:8080
+## Without Docker Compose
 
-Read [more about how to run bblfsh and gitbase dependencies](docs/quickstart.md).
+The playground will run the queries against a [gitbase](https://docs.sourced.tech/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server. Make sure both are properly configured.
 
-### Run with Docker
+Then you can choose to run the Gitbase Playground either as a docker container, or as a binary.
+
+### As a Docker
 
 ```bash
 $ docker pull srcd/gitbase-playground:latest
 $ docker run -d \
     --publish 8080:8080 \
-    --link gitbase \
-    --env GITBASEPG_ENV=dev \
-    --env GITBASEPG_DB_CONNECTION="root@tcp(gitbase:3306)/none?maxAllowedPacket=4194304" \
-    --name gitbasePlayground \
+    --env GITBASEPG_DB_CONNECTION="root@tcp(<gitbase-ip>:3306)/none" \
+    --env GITBASEPG_BBLFSH_SERVER_URL="<bblfshd-ip>:9432" \
     srcd/gitbase-playground:latest
 ```
 
+### As a Binary
 
-### Run the Binary
-
-Download a binary from our [releases section](https://github.com/src-d/gitbase-playground/releases), and run it:
+Download the binary from our [releases section](https://github.com/src-d/gitbase-playground/releases), and run it:
 
 ```bash
 $ /download/path/gitbase-playground
```

Comment:
does it expect any env vars?

Reply:
It's not mandatory if everything works with the default values.

The env vars section already mentions that any execution method accept them. But maybe if we replace this line with:
`$ GITBASEPG_DB_CONNECTION="root@tcp(<gitbase-ip>:3306)/none" GITBASEPG_BBLFSH_SERVER_URL="<bblfshd-ip>:9432" ./gitbase-playground`

It will be more clear that you can use env vars here too. What do you think?

Reply:
Done in 3d4554b